### PR TITLE
Add MySQLi deprecations to `versions.xml`

### PR DESCRIPTION
**Pull request by TimWolla**
see php/doc-en#3958

### CODE REVIEW

File: reference/mysqli/versions.xml [hunk 1]
```diff
@@ -55,22 +55,22 @@
  <function name="mysqli_info" from="PHP 5, PHP 7, PHP 8"/>
  <function name="mysqli_init" from="PHP 5, PHP 7, PHP 8"/>
  <function name="mysqli_insert_id" from="PHP 5, PHP 7, PHP 8"/>
- <function name="mysqli_kill" from="PHP 5, PHP 7, PHP 8"/>
+ <function name="mysqli_kill" from="PHP 5, PHP 7, PHP 8" deprecated="PHP 8.4"/>
```
Comment (kamil-tekiela) [inline reference/mysqli/versions.xml:58]:
```suggestion
 <function name="mysqli_kill" from="PHP 5, PHP 7, PHP 8" deprecated="PHP 8.4.0"/>
```
I think we specify the full version everywhere.

**Review by kamil-tekiela — CHANGES_REQUESTED** [review]
Thanks, I forgot this exists. Could you please mark all 6 of them as deprecated?

File: reference/mysqli/versions.xml (hunk 1 above)
Comment (TimWolla) [reply reference/mysqli/versions.xml:58]: utf8_(de|en)code use the short variant: https://github.com/php/doc-en/blob/0f1d7fba407bd1385616508a1e42e31fe009bff2/reference/strings/versions.xml#L106-L107, but given that the introductory version is named in full this makes sense to me to adjust.

**Comment by TimWolla** [comment]
> Could you please mark all 6 of them as deprecated?

Bah. This duplication between procedural and OO API is terrible. Now adjusted.

**Review by kamil-tekiela — APPROVED** [review]

**Pull request merged**